packages/integrations/cloudflare/src/utils/image-config.ts

```diff
@@ -1,8 +1,10 @@
 import type { AstroConfig, AstroIntegrationLogger, HookParameters } from 'astro';
 import { passthroughImageService, sharpImageService } from 'astro/config';
 
+export type ImageService = 'passthrough' | 'cloudflare' | 'compile' | 'custom';
```

Comment:
Not really relevant for this PR, but added a type here so you'd get the autocomplete when filling the `switch` case in `setImageConfig` and not having separate (less strict) types that mean the same thing.